Kernel/Graphics/Intel/DisplayConnectorGroup.h

```diff
@@ -73,7 +73,6 @@ class IntelDisplayConnectorGroup : public RefCounted<IntelDisplayConnectorGroup>
     void enable_dpll_without_vga(IntelGraphics::PLLSettings const&, size_t dac_multiplier);
     void set_display_timings(DisplayConnector::ModeSetting const&);
     void enable_pipe_a();
-    void set_framebuffer_parameters(size_t, size_t);
```

Comment:
The commit `Kernel/Graphics: Remove stale method in the Intel driver ` should also be able to be squashed into the commit that creates `Kernel/Graphics/Intel/DisplayConnectorGroup.h`